python/ray/serve/api.py

```diff
@@ -734,6 +735,13 @@ def my_deployment_function(request):
     return _request_context.multiplexed_model_id
 
 
+@PublicAPI(stability="beta")
+def get_grpc_context() -> RayServegRPCContext:
```

Comment:
If this returns a `RayServegRPCContext`, and it's a public API, then `RayServegRPCContext` should also be marked a public API since users must interface with it.

In that case, we should also move `RayServegRPCContext` out of the `_private/` subfolder.